### DISCUSSION TITLE
Time to Get Involved!

### DISCUSSION THREAD

**Discussion by kaangiray26** [kaangiray26/geddit-app#51] [Announcements]
Hey guys,

The app is under development for almost a month now and with the release of v1.7 I am quite pleased with the progress so far. I didn't expect to that much of a support and contribution from so many people around the world.

The thing is, I just wanted to make an app in the beginning to give people an alternative for browsing reddit. Because I believe freedom is not just something I fancy but it should be the characteristic of every user out there. And taking it from the people that built the community is something I find very annoying.

Now that the initial phase of the app is over, I begin to realize that Geddit has become something more. It is now not just a simple app but rather a collaborative project to make an app for Reddit with the features that users want and needed.

While I'm so excited for what to come in the future I'll be taking a few weeks off to focus on more simple things. However this does not mean we should stop now.

**Geddit needs you more than ever. Help the project by coming up with new ideas, finding features that are missing, making changes for accessibilities and hoping that it will be awesome in the end.**

Thanks for all the support and love, I really appreciate it.

kaangiray26

**Comment by avincent98144** [comment]
Thanks so much for your efforts Kaan @kaangiray26 .. Here are two additional features I would like to see in all client solutions including LIBREDDIT:

1. Ability to easily FOLLOW or BLOCK a USER set of buttons right next to the REDDIT USER NAME, avoiding the page transfers and several steps to do so currently - awkward and clunky.
2. Remove the current banner overlay that covers all images [ie: "IMAGE" "GIF" etc.] so as not to cover ANY of the actual images. I tried the REMOVE ELEMENT features in the many ADBLOCK, uORIGIN solutions, but have never found them to work globally.

Cheers!

**Comment by brantcy** [comment]
1.7, awesome!  I've been using your app since the day you first announced it on Reddit, never even occurred to me to check for updates...

Been on 1.0 for a couple months now, the new features look awesome!

Great work kaangiray26!
